packages/cli/src/commands/capabilities.ts

```diff
@@ -131,9 +135,9 @@ export interface CapabilityId {
 
 export type CapabilitySummaryWithNamespace = CapabilitySummary & { namespace: string }
 
-export function buildListTableOutput(this: APIOrganizationCommand, capabilities: CapabilitySummaryWithNamespace[]): string {
-	return this.tableGenerator.buildTableFromList(capabilities, ['id', 'version', 'status'])
-}
+// export function buildListTableOutput(this: APIOrganizationCommand, capabilities: CapabilitySummaryWithNamespace[]): string {
+// 	return this.tableGenerator.buildTableFromList(capabilities, ['id', 'version', 'status'])
+// }
```

Comment:
I'm betting you meant to remove this completely after compiling to make sure it wasn't used.

Reply:
I sure did. Thanks.